shared_workflow/install_shared.py

```diff
@@ -236,7 +205,7 @@ def install_simulation(
             else:
                 sim_params_dict.update({key: value})
 
-    return root_params_dict, fault_params_dict, sim_params_dict, vm_params_dict
+    return root_params_dict, fault_params_dict, sim_params_dict
```

Comment:
Line 142/111 (old/new) returns 4 Nones, there should only be 3, or it should just raise an exception